module/read_yaml.py

```diff
@@ -0,0 +1,15 @@
+import yaml
+
+
+class YamlRead(object):
+    def __init__(self, path):
+        self._path = path
+
+    def run(self):
+        yml = dict()
+        with open(self._path, 'r') as stream:
+            try:
+                yml = yaml.safe_load(stream)
+            except yaml.YAMLError as exc:
+                print(exc)
+        return yml
```

Comment:
Filename should start with an uppercase letter and since the class is named YamlRead I'd name the file YamlRead.py too.